dev/com.ibm.ws.webcontainer.security.feature/src/com/ibm/ws/webcontainer/security/feature/internal/FeatureWebSecurityConfigImpl.java

```diff
@@ -413,6 +414,16 @@ public String getSameSiteCookie() {
             return null;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public boolean getPartitionedCookie() {
+        WebAppSecurityConfig globalConfig = WebAppSecurityCollaboratorImpl.getGlobalWebAppSecurityConfig();
+        if (globalConfig != null)
+            return WebAppSecurityCollaboratorImpl.getGlobalWebAppSecurityConfig().getPartitionedCookie();
```

Comment:
```suggestion
            return globalConfig.getPartitionedCookie();
```

Reply:
same thing as before.  that's all over the file so its best to stay consistent.